src/main/java/seedu/easylog/common/Messages.java

```diff
@@ -133,5 +133,9 @@
     public static final String MESSAGE_INVALID_FILE_INPUT_LINE = "Invalid file input at line : ";
     public static final String MESSAGE_ERROR_WHILE_SAVING = "Error while saving. Save file may or may not be "
             + "saved correctly";
+    public static final String  MESSAGE_REPEAT_ITEM = "OOPS!! This item already exist. "
+            + "\nPlease use update function to make modification or use a different name. Thank you!";
+    public static final String MESSAGE_REPEAT_ORDER = "OOPS!! This order already exist. "
+            + "\nPlease use a different name for ordering. Thank you!";
```

Comment:
Can say that if they want to add on to an order that is not done, consider deleting the order first then create.

Reply:
Also can say that if the order is done, and that the particular has a new order, delete also. Just add this for now, I'll decide whether to delete a done order after generating a receipt